src/manage/pages/clients/create-client-dialog.js

```diff
@@ -67,13 +68,13 @@ ClientType.Image = styled.div`
   }
 `
 
-ClientType.Title = styled.div`
-  font-weight: ${fonts.weight.medium};
+ClientType.Title = styled(StyledHeading[4])`
```

Comment:
This is okay as a temporary hack, but if consumers of Cosmos have to alter styles like this, it's a pretty bad developer experience. 😞